torch/random.py

```diff
@@ -15,12 +15,12 @@ def set_rng_state(new_state):
 
 
 def get_rng_state():
-    r"""Returns the random number generator state as a ByteTensor."""
+    r"""Returns the random number generator state as a `torch.ByteTensor`."""
     return default_generator.get_state()
 
 
 def manual_seed(seed):
-    r"""Sets the seed for generating random numbers. And returns a
+    r"""Sets the seed for generating random numbers. This returns a
```

Comment:
Nit: Just "Returns ..." is fine

Reply:
Oops, I forgot to change it in the new commit. Will do it in the next batch.